### DISCUSSION TITLE
pages

### DISCUSSION THREAD

**Discussion by TheOfficalSmurfClub** [community/community#41112] [New to GitHub] [answered]
So I'm brand new to GitHub and I can't get to the settings for pages, it just says add domain but I thought that I don't need to buy a domain if it has .github in it. if anyone knows how to get this to work please tell me.

**Comment by airtower-luna** [comment, marked as answer]
You *automatically* get a `<username>.github.io` domain (or likewise for an organization, if the repo belongs to one), if you want to use that simply don't set a custom domain. Otherwise you have to buy a domain. See also: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites
